### ISSUE TITLE
[Bug] mlc-*-nightly TVM prebuilt package not working on Windows 11.

### ISSUE THREAD

**Issue by Fax** [mlc-ai/mlc-llm#875]
## 🐛 Bug

I am following the instructions (https://mlc.ai/package/) to install the MLC-Chat package to use the REST API on windows.

This is the command:
```pip install --pre --force-reinstall mlc-ai-nightly mlc-chat-nightly -f https://mlc.ai/wheels```

Everything looks good until I try to use it:

```FileNotFoundError: Could not find module 'path\to\site-packages\tvm\tvm.dll' (or one of its dependencies). Try using the full path with constructor syntax.```

The file is there, so there is something else missing. By following the instructions, there is no way to figure out what is missing. 
Either the docs are omitting something or the package has some issues.

I checked then the "Install TVM Unity" under Dependency Installation.
The link to the prebuilt package is the same (https://mlc.ai/package/), so it really doesn't offer any type of insights.

What are the requirements to have TVM running on windows and why the prebuilt package doesn't mention them? 
Which steps are missing?

## To Reproduce

1. follow the instructions for the REST API on a Windows 11 machine.

**Comment by junrushao** [comment]
Could you confirm if tvm.dll exists in the path it mentions? How about tvm_runtime.dll?

**Comment by Fax** [comment]
TL;DR; zstd is the missing library, now everything works.  I just managed to run it without issues.

Long story:
All files are there correctly, the package installed with pip contains the packaged version of tvm.

I have checked the dependencies with dumpbin:
```
Dump of file C:\projects\mlc\venv\Lib\site-packages\tvm\tvm.dll

File Type: DLL

  Image has the following dependencies:

    SHELL32.dll
    ole32.dll
    ADVAPI32.dll
    zstd.dll
    vulkan-1.dll
    KERNEL32.dll
    MSVCP140.dll
    WS2_32.dll
    VCRUNTIME140.dll
    VCRUNTIME140_1.dll
    api-ms-win-crt-runtime-l1-1-0.dll
    api-ms-win-crt-heap-l1-1-0.dll
    api-ms-win-crt-convert-l1-1-0.dll
    api-ms-win-crt-time-l1-1-0.dll
    api-ms-win-crt-math-l1-1-0.dll
    api-ms-win-crt-string-l1-1-0.dll
    api-ms-win-crt-stdio-l1-1-0.dll
    api-ms-win-crt-filesystem-l1-1-0.dll
    api-ms-win-crt-environment-l1-1-0.dll
    api-ms-win-crt-utility-l1-1-0.dll
    api-ms-win-crt-locale-l1-1-0.dll

  Summary

      1C0000 .data
      1F5000 .pdata
     2E9E000 .rdata
       AE000 .reloc
        1000 .rsrc
     43CA000 .text
     
 ```
 
The only lib that I didn't know of was zstd.dll, so I downloaded the precompiled binary from here:
https://github.com/facebook/zstd
(https://github.com/facebook/zstd/releases/tag/v1.5.5)

I copied the libzstd.dll in the same folder as tvm.dll, renamed it zstd.dll and now tvm works correctly.

So it doesn't look like a bug but a dependency of tvm that is not declared anywhere, or maybe I just cannot find any mention of it anywhere in the docs.

**Comment by junrushao** [comment]
Great findings @Fax! The zstd dependency was introduced by LLVM, which TVM depends on. This has been a huge headache to me when doing the windows python packaging, because usually zstd is not present by default on windows, and the attempts to static link zstd into tvm.dll have failed to date :-(

**Comment by Fax** [comment]
I see! It took me too long to figure out what was breaking, but now the system works as expected. Should it be mentioned in the docs until a better solution will be found?

**Comment by junrushao** [comment]
This is very helpful information on my end! I added your findings in our latest doc: https://github.com/mlc-ai/mlc-llm/pull/968. Thanks again ❤️🙏
